api_v3/lib/KalturaRequestDeserializer.php

```diff
@@ -232,6 +232,15 @@ protected function validateParameter($name, &$value, $constraintsObj) {
 			$this->validateMaxValue($name, $value, $constraints[KalturaDocCommentParser::MAX_VALUE_CONSTRAINT]);
 		if(array_key_exists(KalturaDocCommentParser::UTF8_TRUNCATE, $constraints))
 			$this->truncateUTF8String($name,$value, $constraints[KalturaDocCommentParser::UTF8_TRUNCATE]);
+		if ($class && $class === 'KalturaMultiLingualString' && $name === 'language')
+		{
+			$upperCase2charLanguageCode = strtoupper($value);
+			if(!languageCodeManager::getLanguageKey($upperCase2charLanguageCode))
+			{
+				throw new KalturaAPIException(KalturaErrors::INVALID_LANGUAGE_CODE, $value);
+			}
+			$value = $upperCase2charLanguageCode;
```

Comment:
why we need to update the value to upper?

Reply:
the codes in the manager are set in upper. i figured it will look better when returned in the response as upper